### PR TITLE
Add resize for Capsule Arrow

### PR DESCRIPTION
**Pull request by EtaLoop**
Add `resizeCapsule()`, `resizeArrow()` methods

### CODE REVIEW

**Comment by nim65s** [comment]
I did rebase this on the current main branch.
Merging reformats is not trivial, so maybe I made a mistake: Please review this again, @EtaLoop :)

Also, Github shows my profile on those commits after the rebase, and this is an issue, as I am not the author, only the committer. Maybe This is because there is something wrong in your email setup.

Can you double check what `git config --global user.email` says ? If this is not a valid mail, you'll need to set that with eg. `git config --global user.email johndoe@example.fr`

**Comment by EtaLoop** [comment]
I didn't want my mail to show up in my commits back then, so I removed it from `git config --global user.email`.
I only noticed recently that you can choose whether to show your mail or not in github settings.

**Comment by nim65s** [comment]
Ok, even your LAAS email ?
Otherwise, you can trick it and use a github email, `146718995+EtaLoop@users.noreply.github.com` in your case.

**Comment by EtaLoop** [comment]
You are right, it seems that "Keep my email addresses private" in settings only works with web based git operations, not command line git operation.
So setting user.email to 146718995+EtaLoop@users.noreply.github.com will surely work.

**Comment by EtaLoop** [comment]
Looks like the resizing methods are working properly :+1:

File: src/gepetto_viewer_rerun/client.py [hunk 1]
```diff
@@ -109,48 +109,65 @@ def _parse_entity(
...
             for entity in entity_list:
                 if entity.name == entityName:
                     return entity
         return None
 
+    def _isEntityInScene(self, entity: Entity, scene: Scene):
+        if entity and entity.scenes:
+            for s in entity.scenes:
+                if scene == s:
+                    return True
```
Comment (nim65s) [inline src/gepetto_viewer_rerun/client.py:168]:
```suggestion
            return scene in entity.scenes
```

File: src/gepetto_viewer_rerun/entity.py [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
 @dataclass
 class Entity:
+    """
+    Each entity is defined by its name, the archetype and
+        the list of the scenes in which it is drawn.
+    """
+
     name: str
     archetype: rr.archetypes
+    scenes: List[Scene] = None
```
Comment (nim65s) [inline src/gepetto_viewer_rerun/entity.py:17]:
```suggestion
    scenes: List[Scene] | None = None
```

**Pull request merged**